src/integrationTest/java/uk/gov/hmcts/reform/divorce/context/IntegrationTest.java

```diff
@@ -23,12 +24,13 @@
 import java.util.Stack;
 import java.util.UUID;
 import java.util.function.Supplier;
-import javax.annotation.PostConstruct;
 
 @Slf4j
 @RunWith(SerenityRunner.class)
 @ContextConfiguration(classes = {ServiceContextConfiguration.class})
 public abstract class IntegrationTest {
+
+    private static final Object LOCK_OBJECT = new Object();
```

Comment:
I don't think this should be static. In theory each IntegrationTest should be able to make these API calls independently